docs/Vectors.md

```diff
@@ -139,6 +139,46 @@ FT.CREATE ... SCHEMA ... {field_name} VECTOR {algorithm} {count} [{attribute_nam
     EF_RUNTIME 20
     ```
 
-<!-- ## Querying vector fields
+## Querying vector fields
 
-TO BE ADDED -->
+We allow using vector similarity queries in the `FT.SEARCH` "query" parameter. The syntax for vector similarity queries is `*=>[{vector similarity query}]` for running the query on an entire vector field, or `{primary filter query}=>[{vector similarity query}]` for running similarity query on the result of the primary filter query.
+
+As of version 2.4, we allow vector similarity to be used **once** in the query.
+
+* Invalid example: `"(@title:Matrix)=>[KNN 10 @v $B] @year:[2020 2022]"`
+
+* Valid example: `"(@title:Matrix @year:[2020 2022])=>[KNN 10 @v $B]"`
+
+The `{vector similarity query}` part inside the square brackets needs to be in the following format:
+
+    KNN { number | $number_attribute } @{vector field} $blob_attribute [{vector query param name} {value|$value_attribute} [...]] [ AS {score field name | $score_field_name_attribute}]
+
+Every "`*_attribute`" parameter should refer to an attribute in the [`PARAMS`](Commands.md#ftsearch) section.
+
+*   `{ number | $number_attribute }` - The number of results ("K").
+
+*   `@{vector field}` - `vector field` should be a name of a vector field in the index.
+
+*   `$blob_attribute` - An attribute that holds the query vector as blob. must be passed through the `PARAMS` section.
+
+*   `[{vector query param name} {value|$value_attribute} [...]]` - An optional part for passing vector similarity query parameters. Parameters should come in key-value pairs and should be valid parameters for the query. see what [parameters](Vectors.md#specific_attributse_per_algorithm) are valid for each algorithm.
```

Comment:
This is misleading - in [parameters](Vectors.md#specific_attributse_per_algorithm) we specify what are the mandatory and optional parameters upon **creating the index**, not upon searching in it. I think we should add another section under #querying vector that specify the valid parameters for search (which are only EF_RUNTIME for HNSW currently)

Reply:
not misleading, but lacking. Please add a section for runtime parameters per algorithm